### PR TITLE
PlaceListViewController: Initial commit.

### PR DESCRIPTION
**Pull request by codesinc**
This commit adds the PlaceListViewController view controller.
PlaceListViewController is responsible of representing a list of Flickr top places that were downloaded using FlickrTopPlaceDownloader class.

### CODE REVIEW

**Comment by CodeBaseCamp** [comment]
Please update the PR title to something insightful.

**Comment by CodeBaseCamp** [comment]
And fix the PR description, i.e. 
1. Don't copy the commit title. 
2. Use correct punctuation. 
3. Use reasonable linebreaks.

**Comment by CodeBaseCamp** [comment]
Also, your PR description is not complete (it doesn't refer to the first commit of the PR at all).

**Comment by CodeBaseCamp** [comment]
Period at the end of the PR title.

**Comment by CodeBaseCamp** [comment]
And please wait with changes until I'm done with the round.

File: FlickrPhotos/FlickrPhotos/Model/Downloader/FlickrTopPlaceDownloader.m [hunk 1]
```diff
@@ -18,34 +18,72 @@ @interface FlickrTopPlaceDownloader()
...
+        arrayForKey = [[NSMutableArray alloc] init];
+        [arrayForKey addObject:placeMetadata];
+        [topPlaces setObject:arrayForKey forKey:placeMetadata.country];
+      } else {
+        [arrayForKey addObject:placeMetadata];
+        [topPlaces setObject:arrayForKey forKey:placeMetadata.country];
+      }
     }
-    completion([downloadedPlaces copy]);
+    completion([NSDictionary dictionaryWithDictionary:topPlaces]);
```
Comment (CodeBaseCamp) [inline FlickrPhotos/FlickrPhotos/Model/Downloader/FlickrTopPlaceDownloader.m]: See corresponding comment in PR #10.

File: FlickrPhotos/FlickrPhotos/PlaceListViewController.h [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+NS_ASSUME_NONNULL_BEGIN
+
+@class FlickrPlaceMetadata;
+
+/// View controller that updates its table view to be Flickr top places.
+@interface PlaceListViewController : UITableViewController <UISplitViewControllerDelegate>
+
+/// Flickr top places depicted by this instance.
+@property (strong, nonatomic) NSDictionary *places;
```
Comment (CodeBaseCamp) [inline FlickrPhotos/FlickrPhotos/PlaceListViewController.h]: This is not type-safe, but easily can be. Please fix.

File: FlickrPhotos/FlickrPhotos/PlaceListViewController.m [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+                forControlEvents:UIControlEventValueChanged];
+  self.splitViewController.preferredDisplayMode = UISplitViewControllerDisplayModeAllVisible;
+  self.splitViewController.delegate = self;
+}
+
+- (void)downloadTopPlaces {
+  TopPlacesCompletionBlock onComplete = ^(NSDictionary *places) {
+    dispatch_async(dispatch_get_main_queue(), ^{
+      self.sectionTitles =
+      [[places allKeys] sortedArrayUsingSelector:@selector(localizedCaseInsensitiveCompare:)];
```
Comment (CodeBaseCamp) [inline FlickrPhotos/FlickrPhotos/PlaceListViewController.m]: Indentation

File: FlickrPhotos/FlickrPhotos/PlaceListViewController.m [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+}
+
+- (nullable NSString *)tableView:(UITableView *)tableView
+         titleForHeaderInSection:(NSInteger)section {
+  return [self.sectionTitles objectAtIndex:section];
+}
+
+- (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section {
+  NSString *sectionTitle = [self.sectionTitles objectAtIndex:section];
+  NSArray *sectionPlaces = [self.places objectForKey:sectionTitle];
```
Comment (CodeBaseCamp) [inline FlickrPhotos/FlickrPhotos/PlaceListViewController.m]: NSArray of what?

File: FlickrPhotos/FlickrPhotos/PlaceListViewController.m [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+  return sectionPlaces.count;
+}
+
+- (UITableViewCell *)tableView:(UITableView *)tableView
+         cellForRowAtIndexPath:(NSIndexPath *)indexPath {
+  static NSString *cellIdentifier = @"placeCell";
+  UITableViewCell *cell = [tableView dequeueReusableCellWithIdentifier:cellIdentifier];
+  if (cell == nil) {
```
Comment (CodeBaseCamp) [inline FlickrPhotos/FlickrPhotos/PlaceListViewController.m]: `if (cell == nil) {` -> `if (!cell) {`

Comment (CodeBaseCamp) [inline FlickrPhotos/FlickrPhotos/PlaceListViewController.m]: Or shorter:
`cell = cell ?: ...;`

File: FlickrPhotos/FlickrPhotos/PlaceListViewController.m [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+  NSArray *sectionPlaces = [self.places objectForKey:sectionTitle];
+  return [sectionPlaces objectAtIndex:row];
+}
+
+#pragma mark -
+#pragma mark Navigation
+#pragma mark -
+
+- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(_Nullable id)sender {
+  if ([sender isKindOfClass:[UITableViewCell class]]) {
```
Comment (CodeBaseCamp) [inline FlickrPhotos/FlickrPhotos/PlaceListViewController.m]: Bail out fast. :)

File: FlickrPhotos/FlickrPhotos/PlaceListViewController.m [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+    NSIndexPath *indexPath = [self.tableView indexPathForCell:sender];
+    if (indexPath) {
```
Comment (CodeBaseCamp) [inline FlickrPhotos/FlickrPhotos/PlaceListViewController.m]: same

File: FlickrPhotos/FlickrPhotos/PlaceListViewController.m [hunk 8]
```diff
@@ -0,0 +1,146 @@
...
+      if ([segue.identifier isEqualToString:@"place photos"]) {
+        if ([segue.destinationViewController isKindOfClass:[PhotoListViewController class]]) {
```
Comment (CodeBaseCamp) [inline FlickrPhotos/FlickrPhotos/PlaceListViewController.m]: same

File: FlickrPhotos/FlickrPhotos/PlaceListViewController.m [hunk 9]
```diff
@@ -0,0 +1,146 @@
...
+    }
+  }
+}
+
+#pragma mark -
+#pragma mark UISplitViewControllerDelegate
+#pragma mark -
+
+- (BOOL)splitViewController:(UISplitViewController *)splitViewController
+collapseSecondaryViewController:(UIViewController *)secondaryViewController
```
Comment (CodeBaseCamp) [inline FlickrPhotos/FlickrPhotos/PlaceListViewController.m]: Indentation (four whitespaces in this scenario)

File: FlickrPhotos/FlickrPhotos/PlaceListViewController.m [hunk 10]
```diff
@@ -0,0 +1,146 @@
...
+  ontoPrimaryViewController:(UIViewController *)primaryViewController {
+  return YES;
+}
+
+- (nullable UIViewController *)splitViewController:(UISplitViewController *)splitViewController
+separateSecondaryViewControllerFromPrimaryViewController:(UIViewController *)primaryViewController {
```
Comment (CodeBaseCamp) [inline FlickrPhotos/FlickrPhotos/PlaceListViewController.m]: same

**Comment by CodeBaseCamp** [comment]
Round 1.

File: FlickrPhotos/FlickrPhotos/Model/Downloader/FlickrTopPlaceDownloader.m (hunk 1 above)
Comment (codesinc) [inline FlickrPhotos/FlickrPhotos/Model/Downloader/FlickrTopPlaceDownloader.m]: fixed

File: FlickrPhotos/FlickrPhotos/PlaceListViewController.h (hunk 2 above)
Comment (codesinc) [inline FlickrPhotos/FlickrPhotos/PlaceListViewController.h]: fixed

File: FlickrPhotos/FlickrPhotos/PlaceListViewController.m (hunk 3 above)
Comment (codesinc) [inline FlickrPhotos/FlickrPhotos/PlaceListViewController.m]: fixed

File: FlickrPhotos/FlickrPhotos/PlaceListViewController.m (hunk 4 above)
Comment (codesinc) [inline FlickrPhotos/FlickrPhotos/PlaceListViewController.m]: fixed

File: FlickrPhotos/FlickrPhotos/PlaceListViewController.m (hunk 5 above)
Comment (codesinc) [inline FlickrPhotos/FlickrPhotos/PlaceListViewController.m]: fixed

File: FlickrPhotos/FlickrPhotos/PlaceListViewController.m (hunk 6 above)
Comment (codesinc) [inline FlickrPhotos/FlickrPhotos/PlaceListViewController.m]: fixed

File: FlickrPhotos/FlickrPhotos/PlaceListViewController.m (hunk 7 above)
Comment (codesinc) [inline FlickrPhotos/FlickrPhotos/PlaceListViewController.m]: fixed

File: FlickrPhotos/FlickrPhotos/PlaceListViewController.m (hunk 8 above)
Comment (codesinc) [inline FlickrPhotos/FlickrPhotos/PlaceListViewController.m]: fixed

File: FlickrPhotos/FlickrPhotos/PlaceListViewController.m (hunk 9 above)
Comment (codesinc) [inline FlickrPhotos/FlickrPhotos/PlaceListViewController.m]: fixed

File: FlickrPhotos/FlickrPhotos/PlaceListViewController.m (hunk 10 above)
Comment (codesinc) [inline FlickrPhotos/FlickrPhotos/PlaceListViewController.m]: fixed

**Comment by codesinc** [comment]
finished round 2

File: FlickrPhotos/FlickrPhotos/PlaceListViewController.m (hunk 5 above)
Comment (codesinc) [inline FlickrPhotos/FlickrPhotos/PlaceListViewController.m]: fixed PR#14

Comment (CodeBaseCamp) [inline FlickrPhotos/FlickrPhotos/PlaceListViewController.m]: PRs should never contain overlapping code fragments.

File: FlickrPhotos/FlickrPhotos/PlaceListViewController.m [hunk 11]
```diff
@@ -0,0 +1,145 @@
...
+- (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section {
+  NSString *sectionTitle = [self.sectionTitles objectAtIndex:section];
+  NSArray *sectionPlaces = [self.places objectForKey:sectionTitle];
+  return sectionPlaces.count;
+}
+
+- (UITableViewCell *)tableView:(UITableView *)tableView
+         cellForRowAtIndexPath:(NSIndexPath *)indexPath {
+  static NSString *cellIdentifier = @"placeCell";
+  UITableViewCell *cell = [tableView dequeueReusableCellWithIdentifier:cellIdentifier];
```
Comment (CodeBaseCamp) [inline FlickrPhotos/FlickrPhotos/PlaceListViewController.m]: What I meant was 
```
 UITableViewCell *cell = [tableView dequeueReusableCellWithIdentifier:cellIdentifier] ?: 
    [[UITableViewCell alloc] initWithStyle:UITableViewCellStyleSubtitle
                                           reuseIdentifier:cellIdentifier];
```

File: FlickrPhotos/FlickrPhotos/PlaceListViewController.m [hunk 12]
```diff
@@ -0,0 +1,145 @@
...
+    destination.title = [NSString stringWithFormat:@"Photos from %@", chosenPlace.city];
+  }
+}
+
+#pragma mark -
+#pragma mark UISplitViewControllerDelegate
+#pragma mark -
+
+- (BOOL)splitViewController:(UISplitViewController *)splitViewController
+    collapseSecondaryViewController:(UIViewController *)secondaryViewController
```
Comment (CodeBaseCamp) [inline FlickrPhotos/FlickrPhotos/PlaceListViewController.m]: Inconsistent indentation.

File: FlickrPhotos/FlickrPhotos/Model/Downloader/FlickrTopPlaceDownloader.m [hunk 13]
```diff
@@ -18,34 +18,79 @@ @interface FlickrTopPlaceDownloader()
...
   
-  NSURL *topPlacesURL = [FlickrFetcher URLforTopPlaces];
-  [self.downloader retrieveFromURL:topPlacesURL callingCompletion:extractTopPlacesData];
+  [self.downloader retrieveFromURL:[FlickrFetcher URLforTopPlaces]
+                 callingCompletion:extractTopPlacesData];
+}
+
++ (NSDictionary<NSString *, NSArray<FlickrPlaceMetadata *> *> *)deepCopy:(NSMutableDictionary<NSString *,
+                                                                          NSMutableArray<FlickrPlaceMetadata *> *> *)places {
+  NSMutableDictionary<NSString *, NSArray<FlickrPlaceMetadata *> *> *copy =
```
Comment (CodeBaseCamp) [inline FlickrPhotos/FlickrPhotos/Model/Downloader/FlickrTopPlaceDownloader.m]: `copy` isn't a good variable name (due to the usage of the `copy` method).

File: FlickrPhotos/FlickrPhotos/Model/Downloader/FlickrTopPlaceDownloader.m [hunk 14]
```diff
@@ -18,34 +18,79 @@ @interface FlickrTopPlaceDownloader()
...
   };
   
-  NSURL *topPlacesURL = [FlickrFetcher URLforTopPlaces];
-  [self.downloader retrieveFromURL:topPlacesURL callingCompletion:extractTopPlacesData];
+  [self.downloader retrieveFromURL:[FlickrFetcher URLforTopPlaces]
+                 callingCompletion:extractTopPlacesData];
+}
+
++ (NSDictionary<NSString *, NSArray<FlickrPlaceMetadata *> *> *)deepCopy:(NSMutableDictionary<NSString *,
+                                                                          NSMutableArray<FlickrPlaceMetadata *> *> *)places {
```
Comment (CodeBaseCamp) [inline FlickrPhotos/FlickrPhotos/Model/Downloader/FlickrTopPlaceDownloader.m]: Lines must not exceed 100 characters.

Comment (CodeBaseCamp) [inline FlickrPhotos/FlickrPhotos/Model/Downloader/FlickrTopPlaceDownloader.m]: You can use a typedef here to shorten the generic types.

**Comment by CodeBaseCamp** [comment]
Round 2.

File: FlickrPhotos/FlickrPhotos/PlaceListViewController.m (hunk 11 above)
Comment (codesinc) [inline FlickrPhotos/FlickrPhotos/PlaceListViewController.m]: fixed

File: FlickrPhotos/FlickrPhotos/PlaceListViewController.m (hunk 12 above)
Comment (codesinc) [inline FlickrPhotos/FlickrPhotos/PlaceListViewController.m]: fixed

File: FlickrPhotos/FlickrPhotos/Model/Downloader/FlickrTopPlaceDownloader.m (hunk 14 above)
Comment (codesinc) [inline FlickrPhotos/FlickrPhotos/Model/Downloader/FlickrTopPlaceDownloader.m]: fixed

File: FlickrPhotos/FlickrPhotos/Model/Downloader/FlickrTopPlaceDownloader.m (hunk 13 above)
Comment (codesinc) [inline FlickrPhotos/FlickrPhotos/Model/Downloader/FlickrTopPlaceDownloader.m]: fixed

**Comment by codesinc** [comment]
Finished round 3

File: FlickrPhotos/FlickrPhotos/PlaceListViewController.m (hunk 4 above)
Comment (CodeBaseCamp) [inline FlickrPhotos/FlickrPhotos/PlaceListViewController.m]: Not fixed.

File: FlickrPhotos/FlickrPhotos/Model/Downloader/FlickrTopPlaceDownloader.h [hunk 15]
```diff
@@ -5,10 +5,16 @@
...
+
+typedef NSDictionary<NSString *, NSArray<FlickrPlaceMetadata *> *> FlickrPlaceMetadataDictionary;
+
+typedef NSMutableDictionary<NSString *,
+                            NSMutableArray<FlickrPlaceMetadata *> *> MutableFlickrPlaceMetadataDictionary;
 
 /// Completion block that executes in the end of the top places list download.
-typedef void(^TopPlacesCompletionBlock)(NSArray<FlickrPlacePhotoMetadata *> *);
+/// Keys in dictionary represent country name and the values represent places.
+typedef void(^TopPlacesCompletionBlock)(FlickrPlaceMetadataDictionary *);
```
Comment (CodeBaseCamp) [inline FlickrPhotos/FlickrPhotos/Model/Downloader/FlickrTopPlaceDownloader.h]: `NSArray<FlickrPlacePhotoMetadata *> *` <=> `FlickrPlaceMetadataDictionary *` <=> `NSDictionary<NSString *, NSArray<FlickrPlaceMetadata *> *> *`?

Please fix incorrect use of `FlickrPlaceMetadataDictionary` here.

File: FlickrPhotos/FlickrPhotos/Model/Downloader/FlickrTopPlaceDownloader.h [hunk 16]
```diff
@@ -5,10 +5,16 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-@class FlickrPlacePhotoMetadata;
+@class FlickrPlaceMetadata;
+
+typedef NSDictionary<NSString *, NSArray<FlickrPlaceMetadata *> *> FlickrPlaceMetadataDictionary;
```
Comment (CodeBaseCamp) [inline FlickrPhotos/FlickrPhotos/Model/Downloader/FlickrTopPlaceDownloader.h]: Doc

File: FlickrPhotos/FlickrPhotos/Model/Downloader/FlickrTopPlaceDownloader.h [hunk 17]
```diff
@@ -5,10 +5,16 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-@class FlickrPlacePhotoMetadata;
```
Comment (CodeBaseCamp) [inline FlickrPhotos/FlickrPhotos/Model/Downloader/FlickrTopPlaceDownloader.h]: Should stay.

File: FlickrPhotos/FlickrPhotos/Model/Downloader/FlickrTopPlaceDownloader.h [hunk 18]
```diff
@@ -5,10 +5,16 @@
...
+@class FlickrPlaceMetadata;
+
+typedef NSDictionary<NSString *, NSArray<FlickrPlaceMetadata *> *> FlickrPlaceMetadataDictionary;
+
+typedef NSMutableDictionary<NSString *,
```
Comment (CodeBaseCamp) [inline FlickrPhotos/FlickrPhotos/Model/Downloader/FlickrTopPlaceDownloader.h]: Doc

File: FlickrPhotos/FlickrPhotos/Model/Downloader/FlickrTopPlaceDownloader.h (hunk 16 above)
Comment (CodeBaseCamp) [inline FlickrPhotos/FlickrPhotos/Model/Downloader/FlickrTopPlaceDownloader.h]: A typical doc would be
`/// Mapping of countries to corresponding ordered collections of Flickr place metadata.`

**Comment by CodeBaseCamp** [comment]
Round 4.

File: FlickrPhotos/FlickrPhotos/Model/Downloader/FlickrTopPlaceDownloader.h (hunk 15 above)
Comment (codesinc) [inline FlickrPhotos/FlickrPhotos/Model/Downloader/FlickrTopPlaceDownloader.h]: fixed

Comment (codesinc) [inline FlickrPhotos/FlickrPhotos/Model/Downloader/FlickrTopPlaceDownloader.h]: In my opinion, There is no confusion you're  looking on a old version.

File: FlickrPhotos/FlickrPhotos/Model/Downloader/FlickrTopPlaceDownloader.h (hunk 16 above)
Comment (codesinc) [inline FlickrPhotos/FlickrPhotos/Model/Downloader/FlickrTopPlaceDownloader.h]: fixed

File: FlickrPhotos/FlickrPhotos/Model/Downloader/FlickrTopPlaceDownloader.h (hunk 18 above)
Comment (codesinc) [inline FlickrPhotos/FlickrPhotos/Model/Downloader/FlickrTopPlaceDownloader.h]: fixed

File: FlickrPhotos/FlickrPhotos/Model/Downloader/FlickrTopPlaceDownloader.h (hunk 17 above)
Comment (codesinc) [inline FlickrPhotos/FlickrPhotos/Model/Downloader/FlickrTopPlaceDownloader.h]: I was renamed to FlickrPlaceMetadata

**Comment by codesinc** [comment]
Finished round 4 see notes

**Comment by CodeBaseCamp** [comment]
Please rebase on top of the current develop.

**Comment by codesinc** [comment]
Done

File: FlickrPhotos/FlickrPhotos/PlaceListViewController.m [hunk 19]
```diff
@@ -0,0 +1,146 @@
...
+#import "FlickrTopPlaceDownloader.h"
+#import "ImageViewController.h"
+#import "PhotoListViewController.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface PlaceListViewController()
+
+/// Table view section titles.
+@property (strong, nonatomic) NSArray *sectionTitles;
```
Comment (CodeBaseCamp) [inline FlickrPhotos/FlickrPhotos/PlaceListViewController.m]: Generics

File: FlickrPhotos/FlickrPhotos/PlaceListViewController.m (hunk 4 above)
Comment (CodeBaseCamp) [inline FlickrPhotos/FlickrPhotos/PlaceListViewController.m]: Generics

File: FlickrPhotos/FlickrPhotos/PlaceListViewController.m [hunk 20]
```diff
@@ -0,0 +1,146 @@
...
+
+- (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section {
+  NSString *sectionTitle = [self.sectionTitles objectAtIndex:section];
+  NSArray *sectionPlaces = [self.places objectForKey:sectionTitle];
+  return sectionPlaces.count;
+}
+
+- (UITableViewCell *)tableView:(UITableView *)tableView
+         cellForRowAtIndexPath:(NSIndexPath *)indexPath {
+  static NSString *cellIdentifier = @"placeCell";
```
Comment (CodeBaseCamp) [inline FlickrPhotos/FlickrPhotos/PlaceListViewController.m]: Use `static NSString * const kCellIdentfifier = @"placeCell";` and move the line right before the method implementation.

File: FlickrPhotos/FlickrPhotos/PlaceListViewController.m [hunk 21]
```diff
@@ -0,0 +1,146 @@
...
+
+- (BOOL)splitViewController:(UISplitViewController *)splitViewController
+    collapseSecondaryViewController:(UIViewController *)secondaryViewController
+    ontoPrimaryViewController:(UIViewController *)primaryViewController {
+  return YES;
+}
+
+- (nullable UIViewController *)splitViewController:(UISplitViewController *)splitViewController
+    separateSecondaryViewControllerFromPrimaryViewController:(UIViewController *)primaryViewController {
+  if ([(UIViewController *)self.navigationController.visibleViewController
```
Comment (CodeBaseCamp) [inline FlickrPhotos/FlickrPhotos/PlaceListViewController.m]: Why do you need to cast `visibleViewController`? It's already a `UIViewController` according to the iOS specification.

File: FlickrPhotos/FlickrPhotos/PlaceListViewController.m [hunk 22]
```diff
@@ -0,0 +1,146 @@
...
+       isKindOfClass:[PhotoListViewController class]]) {
+    return [[ImageViewController alloc] init];
+  } else if ([(UIViewController *)self.navigationController.visibleViewController
```
Comment (CodeBaseCamp) [inline FlickrPhotos/FlickrPhotos/PlaceListViewController.m]: same

File: FlickrPhotos/FlickrPhotos/PlaceListViewController.h [hunk 23]
```diff
@@ -0,0 +1,18 @@
...
+#import <UIKit/UIKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@class FlickrPlaceMetadata;
+
+/// View controller that updates its table view to be Flickr top places.
+@interface PlaceListViewController : UITableViewController <UISplitViewControllerDelegate>
+
+/// Flickr top places depicted by this instance.
```
Comment (CodeBaseCamp) [inline FlickrPhotos/FlickrPhotos/PlaceListViewController.h]: Explain what happens when you set this property.

**Comment by CodeBaseCamp** [comment]
Round 5.

File: FlickrPhotos/FlickrPhotos/PlaceListViewController.m (hunk 19 above)
Comment (codesinc) [inline FlickrPhotos/FlickrPhotos/PlaceListViewController.m]: fixed

File: FlickrPhotos/FlickrPhotos/PlaceListViewController.m (hunk 20 above)
Comment (codesinc) [inline FlickrPhotos/FlickrPhotos/PlaceListViewController.m]: fixed

File: FlickrPhotos/FlickrPhotos/PlaceListViewController.m (hunk 21 above)
Comment (codesinc) [inline FlickrPhotos/FlickrPhotos/PlaceListViewController.m]: fixed

File: FlickrPhotos/FlickrPhotos/PlaceListViewController.m (hunk 22 above)
Comment (codesinc) [inline FlickrPhotos/FlickrPhotos/PlaceListViewController.m]: fixed

File: FlickrPhotos/FlickrPhotos/PlaceListViewController.h (hunk 23 above)
Comment (codesinc) [inline FlickrPhotos/FlickrPhotos/PlaceListViewController.h]: fixed

**Comment by codesinc** [comment]
finished round 6

**Comment by CodeBaseCamp** [comment]
Squash.

**Comment by codesinc** [comment]
Done

**Pull request merged**